### ISSUE TITLE
Web scraper for degree, minor, option information

### ISSUE THREAD

**Issue by gzcharleszhang** [gzcharleszhang/course-planner#3]

**Comment by infinitecold** [comment]
@gzcharleszhang The Waterloo UGradCalendar websites are pretty unstructured, scraping may not be as easy as I originally thought. Gonna start with just Math faculty plans for this ticket and slowly add on other faculties in the future.

**Comment by gzcharleszhang** [comment]
@infinitecold yeah there are also stuff like CS 3xx - 4xx which is hard to parse directly. I think we should just manually create a json for a BCS, then finish the rest of the functionalities, since this seems hard. Unless you don't mind keep working on this.

**Comment by infinitecold** [comment]
I'm scrapping the web scraping idea, since each faculty seems to have their own way of writing out their plan requirements and it's not feasible to write different scrapers for different faculties. Instead, I'm going to write a script for someone (probably me) to easily input requirements they see on a plan requirements page and it'll automatically convert it into a JSON like in #47.

This would be a lot easier if #6 was done, I'll wait for that first.

cc @gzcharleszhang @voxelpushing

**Comment by gzcharleszhang** [comment]
ahlie @voxelpushing pls do #6 soon to unblock kelvin
